tools/odbclient/tests/test_odbclient.py

```diff
@@ -407,3 +406,83 @@ def test_bigger_file(datapath, abaqus_version, abaqus_bin):
     expected = pd.read_csv(csv_file).set_index(["node_id", "element_id"])
 
     pd.testing.assert_frame_equal(pylife_mesh_tot, expected)
+
+
+def test_determine_abaqus_bin():
+    # This function contains several test cases.
+    # Reason: Setup and teardown is needed, more principled approaches aren't worth the effort here.
+
+    # Setup
+    abaqus_bin_backup = os.environ.get('ODBSERVER_ABAQUS_BIN')
+    Path('mock_abaqus_exe').touch()  # create empty file
+    Path('mock_abaqus_exe_envvar').touch()
```

Comment:
Please use fixtures for that:

https://docs.pytest.org/en/stable/how-to/tmp_path.html
https://docs.pytest.org/en/stable/how-to/fixtures.html#teardown-cleanup-aka-fixture-finalization
https://docs.pytest.org/en/stable/how-to/fixtures.html#fixtures-can-request-other-fixtures